cms/templates/fragments/standalone-page-bootstrap.html

```diff
@@ -0,0 +1,27 @@
+## mako
+
+## Override the default styles_version to use Bootstrap
+<%! main_css = "css/bootstrap/studio-main.css" %>
```

Comment:
Hmmm, having said that, I'm now not so sure. The actual content is very simple. I'll think about this further.